packages/webpack-example-lite/src/index.ts

```diff
@@ -0,0 +1,68 @@
+import {
+    BigNumber,
+    loadMeshStreamingWithURL,
+    Mesh,
+    OrderEvent,
+    SignedOrder,
+    SupportedProvider,
+} from '@0x/mesh-browser-lite';
+
+// tslint:disable:no-console
+(async () => {
+    // Load the WebAssembly bytecode.
+    loadMeshStreamingWithURL('main.wasm');
```

Comment:
I didn't catch this until actually seeing the usage example, but `loadMeshStreamingWithURL` and `loadMeshStreaming` should probably be async functions that return a Promise. Instead, what we're currently doing is:

```
         setImmediate(() => {
                throw err;
            });
```

This can lead to confusing behavior if the WebAssembly bytecode fails to compile for any reason (e.g. the request times out).


We only do this inside of `@0x/mesh-browser` because (1) We want to the WebAssembly to start loading immediately and (2) We're the ones responsible for bundling __main.wasm__ and we bundle it without sending any network requests, which drastically decreases the chances of something going wrong.

Reply:
I'm going to address this in a separate PR.